wiki/osu!coin/en.md

```diff
@@ -0,0 +1,39 @@
+---
+stub: true
+tags:
+  - april fools
+---
+
+# osu!coin
+
+![multiple osu!coins](img/coins2x.png)![a single osu!coin](img/coin2x.png)
+
+The **osu!coin** was a joke, in-game currency mocking other free-to-play monetization schemes. It was introduced for April Fools, 2015, and allowed players to play a single [beatmap](/wiki/Beatmaps/) in exchange for an osu!coin.
+
+## Release
+
+The [official release statement](https://osu.ppy.sh/home/news/2015-03-31-osucoins) explains that the osu!coin was introduced as a way to fund ![][flag_AU] [peppy](https://osu.ppy.sh/users/2)'s private jet as the current monetization scheme wouldn't allow him to buy it in his lifetime. The statement continues, explaining that each new player would start off with 10 osu!coins, and would be given 10 more for each passing day. Existing users would get an additional two osu!coins as an added bonus. Players could then gain more osu!coins by obtaining combo<!--TODO: link to combo multiplier-->. A single osu!coin would be obtained for every 100x combo.
+
+When players would run out of osu!coins they would be unable to play any beatmaps. Though they could purchase more—out of their own will—with the exchange rates:
+
+- 1USD = 1 osu!coin
+- 10USD = 10 osu!coins
+- 999USD = 1000 osu!coins (0.001% savings!)
+- first unborn child = infinite osu!coins (unborn child will work for osu!)
+
+The following day, peppy rolled back the implementation of osu!coins, opened a new subdomain for the osu!web redesign and ended with [this remark](https://osu.ppy.sh/home/changelog/cuttingedge/20150401).
```

Comment:
```suggestion
The following day, peppy rolled back the implementation of osu!coins, opened a new subdomain for the osu!web redesign and ended with a [remark regarding bug fixes and minor improves](https://osu.ppy.sh/home/changelog/cuttingedge/20150401).
```

Reply:
```suggestion
The following day, peppy rolled back the implementation of osu!coins, opened a new subdomain for the osu!web redesign and ended with a [remark regarding bug fixes and minor improves](https://osu.ppy.sh/comments/121803).
```

We can also update the link to peppy's comment. This suggestion is on top of paper-toaster's.